dev/com.ibm.websphere.appserver.beanValidation-2.0/resources/l10n/com.ibm.websphere.appserver.beanValidation-2.0.properties

```diff
@@ -16,4 +16,4 @@
 
 description=The Bean Validation 2.0 specification provides an annotation based model for validating \
  JavaBeans.  It can be used to assert and maintain the integrity of data as it travels through \
- an application.
+ an application. This feature is built on top of the Hibernate Validator Engine.
```

Comment:
I'm not sure we need to capitalize "Engine." We only need to capitalize it here if the word "Engine" is part of the product name, such as how in "WebSphere Application Server," the word "Server" is part of the name. :)

Reply:
correct, in this case `Engine` is part of the name